scripts/makeCustomBuild.mjs

```diff
@@ -0,0 +1,126 @@
+import { rollup } from "rollup";
+import resolve from "rollup-plugin-node-resolve";
+import commonjs from "rollup-plugin-commonjs";
+import babel from "rollup-plugin-babel";
+import { readFile, writeFile } from "fs/promises";
+import { join } from "path";
+import { terser } from "rollup-plugin-terser";
+
+const NPM_PACKAGE_PROD = "NPM_PACKAGE_PROD";
+const ALLOY_COMPONENTS = {
+  ActivityCollector: "ActivityCollector",
+  Audiences: "Audiences",
+  Context: "Context",
+  DataCollector: "DataCollector",
+  EventMerge: "EventMerge",
+  Identity: "Identity",
+  LibraryInfo: "LibraryInfo",
+  MachineLearning: "MachineLearning",
+  Personalization: "Personalization",
+  Privacy: "Privacy"
+};
+const uncommentCode = async () => {
+  let fileContent = await readFile("./src/core/componentCreators.js", "utf-8");
+  // Remove comments from the import statements
+  fileContent = fileContent.replace(/\/\/ import create/g, 'import create');
+  // Remove comments from the export default
+  fileContent = fileContent.replace(/\/\/ create/g, 'create');
+  // Write the uncommented code back to the file
+  await writeFile("./src/core/componentCreators.js", fileContent);
+  console.log("✔️ Removed comments from the import statements and export default");
+};
+
+uncommentCode().catch(error => {
```

Comment:
Why does `uncommentCode()` run twice: once here and once on line 93 in `buildWithComponents()`? It should only run once.

Reply:
Also, the function doesn't do anything because nothing is ever commented.